packages/common/src/common.ts

```diff
@@ -953,6 +961,15 @@ export class Common extends EventEmitter {
     return this._hardfork
   }
 
+  /**
+   * Returns the hardfork excluding the merge hf which has
+   * no effect on the execution capabilities
+   * @returns Hardfork name
```

Comment:
Could you give this a more extensive description? I am starting to "get it" 😋 but I am pretty sure "normal people" won't be able to grasp the concept yet what this method does and in what contexts it can and can't be used.